corehq/pillows/ledger.py

```diff
@@ -37,6 +40,8 @@ def _prepare_ledger_for_es(ledger):
     if not ledger.get('location_id') and ledger.get('case_id'):
         ledger['location_id'] = _location_id_for_case(ledger['case_id'])
 
+    _update_ledger_section_entry_combinations(ledger)
```

Comment:
where will this get called when the pillow get's removed?

Reply:
So my original intention was to totally remove the pillow, but then I found that this is currently how we do daily consumption after running the full test suite. 

My current plan is to keep the pillow, but remove the ES index as that seemed like a good use of the work I had already done.

So in other words, this pillow will stay and it will contain consumption updating and this update for now